katas/content/multi_qubit_gates/preparing_bell_state/Verification.qs

```diff
@@ -9,14 +9,19 @@ namespace Kata.Verification {
     }
 
     operation CheckSolution() : Bool {
-        let isCorrect = VerifyMultiQubitOperation(Kata.BellState, BellState);
+        let solution = Kata.BellState;
+        let reference = BellState;
+        let isCorrect = CheckOperationsEquivalenceOnZeroState(solution, reference, 2);
 
-        // Output different feedback to the user depending on whether the exercise was correct.
-        use target = Qubit[2];
+        // Output different feedback to the user depending on whether the solution was correct.
+        use target = Qubit[2]; // |00〉
```

Comment:
Now that `target` is only used in the else branch below, this use can be moved there.

Reply:
Updated.